docs/data/data-core-editor.md

```diff
@@ -1624,6 +1636,14 @@ Returns an action object used in signalling that the user has begun to type.
 
 Returns an action object used in signalling that the user has stopped typing.
 
+### enterFormattedText
+
+Returns an action object used in signalling that the caret has entered formatted text.
+
+### leaveFormattedText
```

Comment:
Would this be better-named as `exitFormattedText`? It matches `enterFormattedText` more, and the docs confused me at first because I read "left" as in "left/right" not "left/leave".

Reply:
👍 changed this